lib/error.js

```diff
@@ -13,10 +13,20 @@ function FormError(key, options, req, res) {
             return options.message || this.getMessage(key, options, req, res);
         }
     });
+    Object.defineProperty(this, 'title', {
+        enumerable: true,
+        get: function () {
+            return options.title || this.getTitle();
```

Comment:
Indeed, but it looks like the idea was for a subclass to be able to make use of them if it wishes.

I'm not entirely convinced by this approach but I think we should be consistent with what we currently have.